README.md

```diff
@@ -124,12 +124,19 @@ You need a Kubernetes cluster to run against. You can use [k3d](https://k3d.io/)
    kubectl apply -f config/samples/default.yaml
    ```
 
-5. [Optional] For EventMesh backend, set env var **DOMAIN** to the cluster domain in the deployment resource; for example:
+5. [Optional] For EventMesh backend, set the **spec.backend.config.domain** to the cluster public domain in the Eventing Custom Resource; for example:
```

Comment:
As this step is marked as optional, maybe make it clearer why or when you need to add the domain?

Reply:
Updated it